Lib/test/test_support.py

```diff
@@ -1124,6 +1124,33 @@ def inner(*args, **kwds):
         return inner
     return decorator
 
+#=======================================================================
+# Reset locale to C / POSIX locale. In Jython to date the default locale
+# has been Locale.getDefaultLocale(), ie OS determined
+# The new locale.setlocale() support is in beta and off by default,
+# but some tests need the changes it introduces to pass.
+# It requires certain "startup" initialization corresponding to
+# -J-Dpython.locale.control=settable which is hard to simulate in a unit
+# test. To see the calling test pass without the reload throat-clearing,
+# invoke Jython with that parameter on the command line, in either direct or
+# regrtest flavours.
+def reset_locale_toC(initialize=True):
+    import locale
+    if initialize:
+        from datetime import datetime
+        import time
+        from java.lang import System
+        System.setProperty('python.locale.control','settable')
+        import _locale
+        reload(_locale)
+        reload(locale)
+        # Trigger date format cache reload - need lang change
+        import _strptime
+        locale.setlocale(locale.LC_ALL,'de_DE')
```

Comment:
We use comma-space pretty consistently for arguments, PEP-8 style. Please amend here and wherever else you find it. (I noticed it a lot in the scripts embedded in the unit test.) Easy regex-replace perhaps.

Reply:
Ok.